osbs/build/build_requestv2.py

```diff
@@ -113,7 +112,7 @@ def render_custom_strategy(self):
         custom_strategy['from']['name'] = self.user_params.build_image
 
     def render_name(self):
-        """Sets the Build/BuildConfig object name"""
+        """Sets the Build object name"""
```

Comment:
Wait, so we're removing BuildConfigs but keeping Builds?

Reply:
just for a while, to make rewrite to tekton pipelines easier